File: airport_nearest_relevant_airport/v1/get/java/AirportNearest.java
package examples.media.files;

import com.amadeus.Amadeus;
import com.amadeus.Params;
import com.amadeus.exceptions.ResponseException;
import com.amadeus.resources.Location;

public class AirportNearest {

  public static void main(String[] args) throws ResponseException {

    Amadeus amadeus = Amadeus
        .builder("YOUR_API_ID","YOUR_API_SECRET")
        .build();

    // Airport Nearest Relevant (for London)
    Location[] locations = amadeus.referenceData.locations.airports.get(Params
      .with("latitude", 0.1278)
      .and("longitude", 51.5074));

    System.out.println(flightDestinations);
  }
}

File: airport_nearest_relevant_airport/v1/get/iOS SDK/AirportNearest.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// Airport Nearest Relevant (for London)
amadeus.referenceData.locations.airports.get(params: ["longitude": "49.0000",
                                                      "latitude": "2.55"],
                                             onCompletion: {
                                                 response, error in
                                                 if error == nil {
                                                     print(response!.data)
                                                 }
                                        })
